Update Server-Installation.md (#12365)
fleetdm/fleet: docs/Deploying/Server-Installation.md

### Setting up a host

> If you don't have a CentOS host readily available, Fleet recommends using [Vagrant](https://www.vagrantup.com/) for this guide.
> 
> To install vagrant on a mac run `brew install vagrant`. To install vagrant on a pc [download it here](https://developer.hashicorp.com/vagrant/downloads). To install Vagrant on Linux [follow these instructions](https://developer.hashicorp.com/vagrant/downloads).

Reviewer: > I think if we are trying to make the path of least resistance we need to say the words "install vagrant" somewhere in these docs. It does not come bundled with any software on my macbook pro.

@jarodreyes  in that case how about shortening to:
```suggestion
If you don't have a CentOS host readily available, Fleet recommends using [Vagrant](https://www.vagrantup.com/) for this guide. You can find installation instructions on Vagrant's [downloads page](https://developer.hashicorp.com/vagrant/downloads).
```

> To install vagrant on a mac run `brew install vagrant`. To install vagrant on a pc [download it here](https://developer.hashicorp.com/vagrant/downloads). To install Vagrant on Linux [follow these instructions](https://developer.hashicorp.com/vagrant/downloads).

Author: I got blocked on this step, it never mentioned needing to install Vagrant. I think if we are trying to make the path of least resistance we need to say the words "install vagrant" somewhere in these docs. It does not come bundled with any software on my macbook pro.

> Once you have installed Vagrant, run the following to create a vagrant box, start it, and log into it:

Reviewer: Fix capitalization:
```suggestion
Once you have installed Vagrant, run the following to create a Vagrant box, start it, and log into it:
```
